snippets/README.md

```diff
@@ -69,3 +69,46 @@ List of discrete practices to illustrate in snippets:
 - code is rather well formatted and steps of calculation are clear
 - Good error checking
 - Some variable could use more descriptive names (e.g. `doo`)
+
+
+## `code_to_review1.py`
+
+- undocumented, or not properly documented functions
+- lack of descriptiod of expected input / output, returns
+- non-significant naming of the functions
```

Comment:
What do you think of the variables?

Reply:
Should we also consider things like `PEP8` adherence and guidelines in naming things?

My point being: it's certainly a plus on the quality of the (Python) code, esp. when shared openly to other (Pythonistas) to use ?!

Reply:
+1 on names of variable as Thibault was suggesting!

Reply:
yes, that's a good point, altough worth to mention that's it's totally okay to work out a lab convention as long as all members stick to it, I'll add that to the notes

Reply:
I like very much the idea of emphasising this point - as it's worth discussing even among us, in general. I think the point is whether the code should be "internal-convention-compliant" or "external-convention-compliant". 

I guess that if you're aiming at having your code to live in the public, so reviewed and modified by others potential contributors, naming conventions should follow more general guidelines for "external" validity" rather than "internal" (lab) quality.
Nonetheless, I do see similarly possible naming variables like `X` or `W` when it's completely straightforward like in the case of gradient descent example.